fs2/model.py

```diff
@@ -385,6 +396,7 @@ def _validation_batch_idx_0(self, batch, batch_idx, output) -> None:
             vocoder_ckpt = torch.load(
                 self.config.training.vocoder_path, map_location=input_.device
             )
+            # TODO: Should we also wrap this load_hifigan_from_checkpoint in case the checkpoint is not a Vocoder?
```

Comment:
Seems unnecessary here since the cpkt came from the config, not the CLI. But I guess it would not hurt either.